okhttp-logging-interceptor/README.md

```diff
@@ -42,4 +42,4 @@ implementation("com.squareup.okhttp3:logging-interceptor:4.2.1")
 
 
 
- [1]: ../INTERCEPTORS.md
+ [1]: https://square.github.io/okhttp/interceptors/
```

Comment:
Fixed link looks good to me! I'd change the reference from `[1]` to `[interceptors]` (and the corresponding part in `[OkHttp interceptor][interceptors]`).